src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RecordJudgmentCallbackHandler.java

```diff
@@ -88,6 +88,7 @@ private CallbackResponse clearAllFields(CallbackParams callbackParams) {
         caseData.setJoAmountOrdered(null);
         caseData.setJoAmountCostOrdered(null);
         caseData.setJoIsRegisteredWithRTL(null);
+        caseData.setJoSetAsideDate(null);
```

Comment:
Also mark it null in "About to submit" as it will not get deleted when new judgement is entered.

Reply:
Done